scala.html.markdown

```diff
@@ -276,6 +276,7 @@ r foreach println
 // NB: Scala is quite lenient when it comes to dots and brackets - study the
 // rules separately. This helps write DSLs and APIs that read like English
 
+// Why doesn't `println` need any parameters here? Stay tuned for Functional Programming below!
```

Comment:
*I* know that, but at this point in the doc first-class functions haven't been mentioned yet, and all examples thus far have taken parameters. I figured it was useful to at least mention this new thing, if only to say "it will be explained later".